api-contracts/openapi/components/schemas/worker.yaml

```diff
@@ -1,3 +1,18 @@
+WorkerLabel:
+  properties:
+    metadata:
+      $ref: "./metadata.yaml#/APIResourceMeta"
+    key:
+      type: string
+      description: The key of the label.
+    value:
+      type: string
+      description: The value of the label.
+  required:
+    - metadata
+    - key
```

Comment:
Is `value` not required?

Reply:
no, undefined will effectively unset the label